src/main/java/cloud/fogbow/ms/api/http/request/Version.java

```diff
@@ -48,25 +51,34 @@ protected String getVersionNumber(String filePath) throws FileNotFoundException
     }
 
     private String readBuildFromFile(String membershipConfPath) throws FileNotFoundException {
+        InputStream input = loadFileInputStream(membershipConfPath);
         Properties properties = new Properties();
-        InputStream input = new FileInputStream(membershipConfPath);
         String build = "empty";
 
         try {
             properties.load(input);
 
             build = properties.getProperty(ConfigurationPropertyKeys.BUILD_NUMBER_KEY,
                     ConfigurationPropertyDefaults.BUILD_NUMBER);
-        } catch (IOException e) {
+        } catch (Exception e) {
             LOGGER.warn(String.format(Messages.Warn.ERROR_READING_CONF_FILE, membershipConfPath), e);
         } finally {
             try {
                 input.close();
-            } catch (IOException e) {
+            } catch (Exception e) {
                 LOGGER.warn(String.format(Messages.Warn.ERROR_CLOSING_CONF_FILE, membershipConfPath), e);
             }
         }
-
         return build;
     }
+
```

Comment:
why this method throws an exception if it is treated? That applies to the methods above as well.